app/src/main/res/values-en/strings.xml

```diff
@@ -1377,4 +1377,106 @@
     <string name="menu_teachers">Teachers</string>
     <string name="edziennik_progress_endpoint_addressbook">Syncing addressbook…</string>
     <string name="send_message">Send message</string>
+    <string name="color_black">Black</string>
+    <string name="color_grey">Gray</string>
+    <string name="card_type_notes">Notes</string>
+    <string name="menu_notes">Notes</string>
+    <string name="notes_editor_dialog_title">Edit note</string>
+    <string name="notes_editor_color">Color</string>
+    <string name="color_brown">Brown</string>
+    <string name="color_pink">Pink</string>
+    <string name="color_dark_blue">Dark blue</string>
+    <string name="color_purple">Purple</string>
+    <string name="color_blue">Blue</string>
+    <string name="color_teal">Teal</string>
+    <string name="color_green">Green</string>
+    <string name="color_yellow">Yellow</string>
+    <string name="color_orange">Orange</string>
+    <string name="color_red">Red</string>
+    <string name="notes_action_add">Add note</string>
+    <string name="notes_type_grade">Grade</string>
+    <string name="error_62_reason">No internet connection</string>
+    <string name="error_5003_reason">API is unavailable</string>
+    <string name="error_301_reason">Incorrect symbol</string>
+    <string name="error_302_reason">Incorrect token</string>
+    <string name="error_920_reason">An error occured</string>
+    <string name="error_930_reason">Failed to download file from OneDrive</string>
+    <string name="error_904_reason">Report Error: Exception in API LIBRUS®</string>
+    <string name="privacy_policy_dialog_html"><![CDATA[By using the application, you confirm that you have <a href = "https://szkolny.eu/privacy-policy">read the Privacy Policy</a> and accept its provisions.<br /><br />The authors of the application are not responsible for the use of the Szkolny.eu application.]]></string>
+    <string name="login_chooser_version_format">Szkolny.eu v%s %s</string>
```

Comment:
Didn't you miss a linebreak here?

Reply:
No, I didn't

Reply:
![obraz](https://user-images.githubusercontent.com/30433568/139597305-4ffa01d2-5683-48d9-a6e6-f6dbe1247219.png)
Yes, you did.

Reply:
Oh yeah, you're right